int8 SpatialBN

jspark1105 opened the pull request:
Summary: SpatialBN int8 version

Differential Revision: D13971224

shahzadlone commented on caffe2/quantization/server/spatial_batch_norm_dnnlowp_op.cc:
```diff
@@ -0,0 +1,141 @@
+template <typename T>
+bool SpatialBNDNNLowPOp<T>::RunOnDevice() {
+  const auto& X = InputTensorCPU_(INPUT);
+  const auto& scale = Input(SCALE);
```
I appreciate your uses of ```const```

jspark1105 committed:
int8 SpatialBN (#16796)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/16796

SpatialBN int8 version

Reviewed By: dskhudia

Differential Revision: D13971224

fbshipit-source-id: 0f1b725925bbf2e9edde67b75fd3e2bab0957d4c
